app/src/main/java/org/imozerov/streetartview/ui/detail/DetailArtObjectActivity.kt

```diff
@@ -104,10 +111,10 @@ class DetailArtObjectActivity : AppCompatActivity() {
                     val uriBegin = "geo: $latitude, $longitude"
                     val query = "$latitude, $longitude ($label)"
                     val encodedQuery = Uri.encode(query)
-                    val uriString = "$uriBegin?q=$encodedQuery&z=16";
-                    val uri = Uri.parse(uriString);
-                    val intent = Intent(android.content.Intent.ACTION_VIEW, uri);
-                    startActivity(intent);
+                    val uriString = "$uriBegin?q=$encodedQuery&z=16"
```

Comment:
We now can use anko for this:
http://blog.jetbrains.com/kotlin/2015/05/advanced-features-of-anko/